<launch> 
    <include file="$(find openni2_launch)/launch/openni2.launch">
        <arg name="depth_registration" value="true"/>
    </include>
    <node name="RGBD_hog" pkg="dlib" type="RGBD_hog" output="screen">
    	<param name="min1" value="0.5"/>
    	<param name="max1" value="5.5"/>
    	<param name="min2" value="10.0"/>
    	<param name="max2" value="15.5"/>
    	<param name="scale_x" value="2"/>
    	<param name="scale_y" value="2"/>
    	<param name="adj_thresh" value="-0.2"/>
        <param name="depth_im_topic" value="/camera/depth/image"/>
        <param name="rgb_im_topic" value="/camera/rgb/image_raw"/>  
        <param name="svm_path" value="/home/miningrox/Workspaces/test_ws/src/fused_people_detection/dlib/detector/face_detector.svm"/>    
    </node>
</launch>